src/frontend/webpack.config.js

```diff
@@ -0,0 +1,100 @@
+const path = require("path");
+const HtmlWebPackPlugin = require("html-webpack-plugin");
+const webpack = require("webpack");
+
+require("dotenv").config({ path: "./.env" });
+
+module.exports = (env) => ({
```

Comment:
I added this `env` argument here, so that in the next issue/task, different `.env` files should be pulled depending on if it is in dev, stg, or prd- for example, we may have `.env.dev`, `.env.staging`, `.env.production`, etc. And then we should use this `env` argument to see which `.env.<something>` to use, before create a build.

Reply:
Yes. This sounds like a great approach to me!